vrs/vrs_annotation_batch.py

```diff
@@ -332,12 +310,12 @@ def main(args):
     )
     parser.add_argument(
         "--working-bucket",
-        help="Name of GCP Bucket to mount for access.",
+        help="Name of GCP Bucket to output intermediate files (sharded VCFs and checkpointed HTs) to. Final outputs always go in gs://gnomad-vrs-io-finals/ht-outputs/ .",
```

Comment:
"Final outputs always go in gs://gnomad-vrs-io-finals/ht-outputs/ " --> final outputs for tests go to working bucket